src/api/sfc-spec.md

```diff
@@ -136,6 +136,13 @@ Beware that `src` imports follow the same path resolution rules as webpack modul
 </unit-test>
 ```
 
+:::warning Note
+While using aliases in `src`,don't start with `~`(It can be used in `import` seq ), anything after it is interpreted as a module request. This means you can reference assets inside node modules:
```

Comment:
```suggestion
While using aliases in `src`, don't start with `~`(it can be used in `import` seq ), anything after it is interpreted as a module request. This means you can reference assets inside node modules:
```

I think we need more clarification on `import seq`, it doesn't sound clear enough for new developers

Reply:
Should we add a code block to explain?
Like this:
```vue
// use as alias `~`
import XXX from "~/X"
```

Reply:
Honestly, I feel like skipping it altogether will reduce the confusion 😅 We're speaking about `src` here so let's not mention global imports. I'll leave a suggestion

Reply:
Ok, I'll create a new pr to clarify `import` behave in [`<script setup>`](https://vuejs.org/api/sfc-script-setup.html).